R/multitables.R

```diff
@@ -85,25 +85,32 @@ setMethod("is.public<-", "Multitable", function (x, value) {
 #' name(m) # [1] "gender + age4 + marstat"
 #' }
 #' @export
-newMultitable <- function (formula, data, name, ...) {
-    ## Validate inputs
-    if (missing(formula)) {
-        halt("Must provide a formula")
-    }
-    if (missing(data) || !is.dataset(data)) {
-        halt(dQuote("data"), " must be a Dataset")
-    }
+newMultitable <- function (formula, data, name, multitable=NULL, ...) {
+
+    if (missing(multitable)){
```

Comment:
Since adding this argument turns the whole function into an if-else, let's just make it a different function. Call it `importMultitable` or something. 

Don't forget to document it and export it from the namespace.